### ISSUE TITLE
Update install instructions

### ISSUE THREAD

**Issue by mcflugen** [landlab/landlab#440]
I think we need to update our installation instructions. Currently they instruct users to use a combination of `pip` and `conda` to install everything. Since we're telling users to use `conda` for some steps, we might as well just stick with `conda` for all the steps. If we want to help users who don't want to use Anaconda, we should have a separate set of instructions that just use `pip`.

## Regular install with conda

    $ conda install landlab -c landlab

## Developer install with conda

    $ conda install --file=requirements.txt
    $ python setup.py develop

I'm not exactly sure what the equivalent `pip` instructions would be. It would mostly just be `pip install ...` except that (I don't think) the `netcdf4` package is offered on PyPI, which means users would have to build it themselves.

Do the above instructions work for folks? Do we need equivalent `pip` instructions? Are we find sticking with `conda`?

**Comment by SiccarPoint** [comment]
This makes total sense to me.

**Comment by gregtucker** [comment]
me too, thanks for catching this @mcflugen

**Comment by BenjiDa** [comment]
Yes, this works in a conda virtual environment. Although for some reason it is not working with ipython, see [here](https://github.com/landlab/landlab/issues/441).
